docs/mockups/background.html

```diff
@@ -1658,6 +1664,32 @@ <h4 class="modal-title">
         $("#delete-form").remove();
     }
 
+    const infoMessage = "@@INFO_MESSAGE@@";
```

Comment:
This section from line 1667 to line 1692 is duplicated in around 40 HTML files. Would it make sense to pull it out in to an external JS script, and avoid all the duplication?

Reply:
Thanks for the feedback. Unfortunately, this can't be pulled out of the HTML files because the @@ variables are placeholders that the mumps parser will replace. That parser only works on individual HTML pages at a time, and has no affect on included JS files.

Reply:
Of course. That makes perfect sense. Bummer that you have to keep duplicating all that code. Thanks for the explanation.

Reply:
I don't actually. There is no duplicated code anywhere. All the source code for these HTML files are in the jinja2 templates. The HTML files are generated (see build.sh). 😉

Reply:
Ah, that also makes sense. Thanks for the info.

Reply:
Ah, that also makes sense. Thanks for the info.